manim/utils/space_ops.py

```diff
@@ -178,15 +181,15 @@ def rotation_matrix(angle, axis):
     return reduce(np.dot, [z_to_axis, about_z, axis_to_z])
 
 
-def rotation_about_z(angle):
+def rotation_about_z(angle: Union[int, float]) -> List[Union[int, float]]:
```

Comment:
https://docs.manim.community/en/latest/contributing/documentation.html#adding-type-hints-to-functions-and-parameters Union[int, float]) to float